### DISCUSSION TITLE
How do I have an error handler that's only used for a specific endpoint?

### DISCUSSION THREAD

**Discussion by TheTechRobo** [pallets/flask#4178] [Q&A] [answered]
There is a specific part of my site that should get a specific erorr handler. However, Nowhere in Flask's documentation I can find  how to do this. Is there a way ?

**Comment by davidism** [comment, marked as answer]
Use a blueprint for those routes and error handler.

**Reply by TheTechRobo** [reply]
That worked, thanks. How would I pass on the variables used in the endpoint e.g. with `@bprint.route('/foo/<bar>')` how would `bar` be passed onto the error handler?

**Reply by ThiefMaster** [reply]
`request.view_args` is a dict containing all those variables

**Reply by TheTechRobo** [reply]
Thanks all!
